shared/fs/common/errs.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import features from '../../util/feature-flags'
+
+type ErrProps = {
+  dismiss: () => void,
+  error: string,
+  msg: string,
+  onFeedback?: () => void,
+  retry?: () => void,
+  time: number,
+}
+
+const ErrToBanner = (props: ErrProps) => (
+  <Kb.Banner
+    onClose={props.dismiss}
+    text={props.msg}
+    color="red"
+    actions={[
+      ...(props.retry ? [{onClick: props.retry, title: 'Retry'}] : []),
+      ...(props.onFeedback ? [{onClick: props.onFeedback, title: 'Let us know'}] : []),
+    ]}
+  />
+)
+
+const Err = features.admin
```

Comment:
why this for admin?

Reply:
It's for displaying error details, so that we don't have to pull up the logs to see which action actually happened. On the other hand user doesn't need to see it.

Reply:
you'll see this error in the console etc right? i dislike having us only see some special version of something in the app and not seeing what the user sees

Reply:
Yup. OK; I'll just remove the tooltip thing.